src/Effects/Tremolo.js

```diff
@@ -0,0 +1,83 @@
+Pizzicato.Effects.Tremolo = function(options) {
+
+	this.options = {};
+	options = options || this.options;
+
+	var defaults = {
```

Comment:
If you can, have a look at this branch and let me know. https://github.com/JohnPaulHarold/pizzicato/tree/tremolo2 I can merge that into the current branch and we can resume the review with that instead.